pkg/skaffold/schema/latest/config.go

```diff
@@ -497,6 +497,9 @@ type TestCase struct {
 type DeployConfig struct {
 	DeployType `yaml:",inline"`
 
+	// StatusCheck *beta* enables waiting for deployments to stabilize.
+	StatusCheck bool `yaml:"statusCheck,omitempty"`
```

Comment:
I think `omitempty` only works if this is `*bool`

Reply:
also, this needs to be `*bool` anyways. Otherwise it'll get a value of `false` by default and you can't distinguish if that value was set by the user or not (same problem as with the `status-check` flag that you just fixed).

Reply:
👍  Updated! (My initial implementation had `skaffold fix` automatically add a `StatusCheck: true` stanza to the skaffold.yaml to preserve the existing enabled-by-default behavior, so I also removed that logic.)